_api-reference/cluster-api/cluster-reroute.md

```diff
@@ -0,0 +1,239 @@
+
+| Parameter        | Data type | Description                                                                                        |
+| ---------------- | --------- | -------------------------------------------------------------------------------------------------- |
+| `dry_run`        | Boolean   | If `true`, validates and simulates the reroute request without applying it. Default is `false`.    |
+| `explain`        | Boolean   | If `true`, returns an explanation of why the command was accepted or rejected. Default is `false`. |
+| `retry_failed`   | Boolean   | If `true`, retries allocation of shards that previously failed. Default is `false`.                |
+| `metric`         | String    | Limits the returned metadata. See [metric options](#metric-options) for list of available options. Default is `_all` |
+| `master_timeout` | Time      | The timeout for connection to the cluster manager node. Default is `30s`.                              |
+| `timeout`        | Time      | The overall request timeout. Default is `30s`.                                                         |
+
+### Metric options
+
+The `metric` parameter narrows the parts of the cluster state returned by the reroute API. This is useful for reducing response size or inspecting specific parts of the cluster state. The possible options are the following:
+
+- `_all`: Returns all available cluster state sections. _(Default)_
+- `blocks`: Includes information about read/write level blocks in the cluster.
+- `master_node`: Shows which node is currently acting as cluster manager.
+- `metadata`: Returns index settings, mappings, and aliases. If specific indexes are targeted, only their metadata is returned.
+- `nodes`: Includes all nodes in the cluster and their metadata.
+- `routing_table`: Returns the routing information for all shards and replicas.
+- `version`: Displays the cluster state version number.
+
+You can combine values in a comma-separated list, such as `metric=metadata,nodes,routing_table`.
+
+## Request body fields
+
+The `commands` array in the request body defines actions to apply to shard allocation. Supported actions include:
+
+### Move
+
+Moves a started shard (primary or replica) from one node to another. This can be used to balance load or drain a node before maintenance. The shard must be in the `STARTED` state. Both primary and replica shards can be moved using this command. 
+
+`move` command requires the following parameters:
+
+* `index`: The name of the index.
+* `shard`: Shard number.
+* `from_node`: Name of the node to move the shard from.
+* `to_node`: Name of the node to move the shard to.
+
+### Cancel
+
+Cancels allocation of a shard (including recovery). This command forces resynchronization by canceling existing allocations and letting the system reinitialize them. Replica shard allocations can be canceled by default, but canceling a primary shard requires `allow_primary=true` to prevent accidental data disruption
```

Comment:
```suggestion
The `cancel` command cancels allocation of a shard (including recovery). This command forces resynchronization by canceling existing allocations and letting the system reinitialize them. Replica shard allocations can be canceled by default, but canceling a primary shard requires `allow_primary=true` to prevent accidental data disruption
```